### DISCUSSION TITLE
Can't import conf file because of an error

### DISCUSSION THREAD

**Discussion by safe-bug** [shadowsocks/shadowsocks-android#2978] [Q&A]
Hello

I have a Sony TV with Android 11 when I want to touch the "Replace from file..." option it says "You don't have an app that can do this". I don't see any permission on the list to give to Shadowsocks and the error doesn't say that it needs what kind of application. What is the problem?

Thanks

**Comment by mdbaniani** [comment]
I guess u can do this after installing  file commander app. the thing that I don't know is what kind of file it supports
